index.html

```diff
@@ -10,43 +10,52 @@
 </head>
 
 <body>
-    <main class="main-section">
 
         <header>
             <nav>
+                <ul class="top-nav">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
                 <img src="images/burgers.svg" alt="Menu-bar" class="hamburgers">
             </nav>
         </header>
+        <main class="main-section">
 
-        <section>
-            <div class="commence">
-                <h1 class="Introduction">
-
-                    Hey there. <br class="break">
-                    I'm Young. <br>
-                    <span class="green-h1">I'm a Software <br class="break">Developer</span>
-                </h1>
-                <div class="d-first-paragraph">
-                    <p class="first-paragraph"> I can help you build a product, feature or website Look through some of
-                        my work
-                        and experience! If you like what you see and have a project you need coded,
-                        don’t hesitate to contact me.</p>
-                </div>
+        <!-- <section> -->
+        <div class="commence">
+            <h1 class="Introduction">
+
+                Hey there. <br class="break">
+                I'm Young. <br>
+                <span class="green-h1">I'm a Software <br class="break">Developer</span>
+
+            </h1>
+
+            <div class="d-first-paragraph">
+
+                <p class="first-paragraph"> I can help you build a product, feature or website Look through some of
+                    my work
+                    and experience! If you like what you see and have a project you need coded,
+                    don’t hesitate to contact me.
+                </p>
```

Comment:
- [ ]  I would kindly suggest you to make your paragraph text content to be align to avoid overlap with the background images, kindly align the text to fit the original figma design, this will make you text look readable and fit to the design kindly fix this .
- [ ]  your design

![Screenshot 2023-06-15 185031](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/80469e30-a503-4535-ba7e-ad029d4186a0)


- [ ] figma desig 
![fig design3](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/7b0c5eff-e821-45ee-9aa6-72c5823f3a48)